### ISSUE TITLE
Drop Clojure 1.2 and 1.3 support

### ISSUE THREAD

**Issue by michaelklishin** [clj-time/clj-time#139]
I'd like to propose dropping 1.2 and 1.3 support. We've been supporting them for a while after they've stopped being widely used. I can't name many projects that try to support 1.2 any more.

We've started running into issues with dependencies not supporting 1.2/1.3 and the need
to carefully separate and manage Leiningen profiles to get things running (`midje-readme` just took an hour of my time).

Bottom line: we are one of the last projects out there to support 1.2 and 1.3 doesn't matter much either. Supporting 3 most recent releases is an OK policy if you ask me.
